app.js

```diff
@@ -13,5 +13,423 @@ menuIcon.onclick = () => {
+  liveLink.innerHTML = 'See live&nbsp;&nbsp;<i class="uil uil-export"></i>';
+  liveLink.href = projectCardInfo.liveLink;
+  popupLinkContainer.appendChild(liveLink);
+
+  const sourceLink = createElement('a', 'sourceLink');
+  sourceLink.innerHTML = 'See source&nbsp;&nbsp;<i class="uil uil-github"></i>';
+  sourceLink.href = projectCardInfo.sourceLink;
+  popupLinkContainer.appendChild(sourceLink);
+
+  projectPopup.appendChild(docFragment);
+  popupContainer.appendChild(projectPopup);
+  body.appendChild(popupContainer);
+
+  document.querySelector('#bar').style.filter = 'blur(10px)';
+
+  document.querySelector('#firstsection').style.filter = 'blur(10px)';
+  document.querySelector('#about-me').style.filter = 'blur(10px)';
+
+  const closeBtn = document.getElementById('closeBtn');
+  closeBtn.addEventListener('click', () => {
+    body.removeChild(popupContainer);
+    document.querySelector('#bar').style.filter = 'blur(0)';
+
+    document.querySelector('#firstsection').style.filter = 'blur(0)';
+    document.querySelector('#about-me').style.filter = 'blur(0)';
+  });
+}
+
+const projectsBtn = document.querySelectorAll('.project-btn');
+projectsBtn.forEach((btn) => btn.addEventListener('click', projectModal));
+const projectsBtn2 = document.querySelectorAll('.work1Link');
+projectsBtn2.forEach((btn) => btn.addEventListener('click', projectModal));
+
+// if (projectsArray[i] === projectMob1 ){
+//   constTop.classList.add('modalTopParent');
+//   body.append(modalTop);
+
+//   //=======get modal Info=======
+//   const modalTopInfo = projectsArray [0];
+//   const {
+//     imageTop,titleTop ,descTop,techTop,liveTop,sourceTop
+//   } = modalTopInfo;
+//   //======= add the to modal
+//  //====== top modal child section
+//   const modalTopChild = document.createElement('section');
+//   modalTopChild.classList.add('modalSection');
+//   modalTop.appendChild(modalTopChild);
+
+//   //=======get modal Info=======
+//   const modalTopInfo = projectsArray [0];
+//   const {
+//     imageTop,titleTop ,descTop,techTop,liveTop,sourceTop
+//   } = modalTopInfo;
+//   //======= add the to modal
+//  //====== top modal child section
+//   const modalTopChild = document.createElement('section');
+//   modalTopChild.classList.add('modalSection');
+//   modalTop.appendChild(modalTopChild);
+
+//   // top modal heading section
+//   const modalHeading = document.createElement('article');
+//   modalHeading.classList.add('heading-section');
+
+//   //========== top modal heaad-text
+// const heading = document.createElement('h2');
+// heading.classList.add('modalH2');
+// heading.innerHTML = titleTop;
+
+// // ===top modal x button
+// const Xbutton = document.createElement('span');
+// Xbutton.classList.add("Xbutton");
+// Xbutton.innerHTML = '&times;';
+// //====  appending element heading button =>article =>topmodalchild
+// modalHeading.append(heading, Xbutton);
+// modalTopChild.append(modalHeading);
+
+// console.log(modalHeading)
```

Comment:
You have done a perfect work, however, we have a few points of improvement, talking about line comment, it helps us clean and standardizes our code regarding best practices, to say so, I believe it is not a good practice to add them in our code, I would strongly recommend you remove them, this will contribute to your professionalism in the near future, what do you think about it? 😃